### DISCUSSION TITLE
orderByToken query is deprecated, what is the replacement ?

### DISCUSSION THREAD

**Discussion by pitkes22** [saleor/saleor#13257] [Q&A] [answered]
I want to create a permalink for each order that will also work for orders created anonymously. I found the `orderByToken` query that allows me to achieve exactly this [but it is deprecated](https://docs.saleor.io/docs/3.x/api-reference/orders/queries/order-by-token), is there a new way to accomplish this?

**Comment by timuric** [comment, marked as answer]
Hi @pitkes22 you can use id for fetching orders for anonymous orders
